src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/WindowsRuntime/Generated/WinRT/Marshalers.cs

```diff
@@ -127,7 +127,7 @@ public void Dispose()
                         marshaler?.Dispose();
                     }
                 }
-                if (_array != null)
+                if (_array != IntPtr.Zero)
```

Comment:
This file is in a "Generated" folder, implying that this code was written (or passed through) by a tool.  We should notify the owner of the tool (or the underlying code template), to fix this and the other two instances in the "original".  I'm not sure who that is - this code came from outside the WPF team fairly recently (it's not in .NETFx).

Reply:
@jkoritzinsky:  Can we modify this CSWinRT file directly?  It looks like it came in with issue #3092: https://github.com/dotnet/wpf/pull/3635.

Reply:
Yes, you can modify these files as needed. And good catch! This is a bug in C#/WinRT.

Reply:
Thanks.